### ISSUE TITLE
varmap() seeps into `nix-shell`

### ISSUE THREAD

**Issue by Profpatsch** [target/lorri#97]
If you `cd lorri` (this project) with the direnv integration, run `lorri watch` and wait for one build to finish, refresh the direnv environment and then try to open a `nix-shell` (over the direnv environment`), the following error happens:

```
> nix-shell
bash: /nix/store/dj3wnbq277ilxc1wxh95pp7d6r035gxq-lorri/varmap-v1: No such file or directory
```

So somehow the code from `src/ops/direnv/envrc.bash` is executed when opening the `nix-shell` in `lorri`?!

A `nix-shell --pure` succeeds, so it depends on the `direnv` environment.

**Comment by manveru** [comment]
The current fix for this seems to be `unset preHook` in the `shellHook`.

**Comment by grahamc** [comment]
This is resolved from #127.
